source/Irrlicht/CBAWMeshFileLoader.h

```diff
@@ -51,16 +51,22 @@ class CBAWMeshFileLoader : public IMeshLoader
 			for (std::map<uint64_t, void*>::iterator it = createdObjs.begin(); it != createdObjs.end(); ++it)
 				loadingMgr.releaseObj(blobs[it->first].header->blobType, it->second);
 		}
-		void regularDrop();
+		void releaseAllButMesh()
+		{
+			for (std::map<uint64_t, void*>::iterator it = createdObjs.begin(); it != createdObjs.end(); ++it)
+			{
+				const uint32_t t = blobs[it->first].header->blobType;
+				if (t != core::Blob::EBT_MESH && t != core::Blob::EBT_SKINNED_MESH)
```

Comment:
no.... just check against the specific blob header's data you are returning

(make the function take the createObjs iterator of the blob that you chose as the mesh to load)